spec/classes/server_spec.rb

```diff
@@ -9,39 +9,37 @@
       describe 'when using default params' do
         it {
           is_expected.to contain_class('xinetd')
-          is_expected.to contain_xinetd__service('rsync').with({ 'bind' => '0.0.0.0' })
+          is_expected.to contain_xinetd__service('rsync').with(bind: '0.0.0.0')
           is_expected.not_to contain_service('rsync')
           is_expected.not_to contain_file('/etc/rsync-motd')
-          is_expected.to contain_concat__fragment('rsyncd_conf_header').with({
-            :order => '00_header',
-          })
-          is_expected.to contain_concat__fragment('rsyncd_conf_header').with_content(/^use chroot\s*=\s*yes$/)
-          is_expected.to contain_concat__fragment('rsyncd_conf_header').with_content(/^address\s*=\s*0.0.0.0$/)
-          is_expected.to contain_concat__fragment('rsyncd_conf_header').with_content(/^pid file\s*=\s*\/var\/run\/rsyncd.pid$/)
+          is_expected.to contain_concat__fragment('rsyncd_conf_header').with(order: '00_header')
```

Comment:
I'd rather keep updates that change functionality in a separate PR

Reply:
This is not updating functionality but just updating how a dict is formatted.
I can separate this but then this PR doesn't pass the lint tests by RuboCop.

Reply:
oh, I totally missed the other line because it looked so different. haha! 👍